src/Tabs/TabTemplate.js

```diff
@@ -11,6 +11,7 @@ const TabTemplate = ({children, selected, style}) => {
   if (!selected) {
     templateStyle.height = 0;
     templateStyle.overflow = 'hidden';
+    templateStyle.display = 'none';
```

Comment:
allows to add a position fixed or position absolute to the content of the tab, and it will hide the content on switching tabs now.